From 93614ce4a9faa0a554f28c5a5f65537a8e6c5ad7 Mon Sep 17 00:00:00 2001
From: Dengke Du <pinganddu90@gmail.com>
Date: Wed, 10 Sep 2025 17:26:36 +0800
Subject: [PATCH] SuiteSparse was confirm to build with ceres

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6bc4e7dc6..a1554993a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,7 +13,7 @@ find_package(nav2_util REQUIRED)
 find_package(nav2_msgs REQUIRED)
 find_package(nav2_costmap_2d REQUIRED)
 find_package(pluginlib REQUIRED)
-find_package(Ceres REQUIRED COMPONENTS SuiteSparse)
+find_package(Ceres REQUIRED)
 
 set(CMAKE_CXX_STANDARD 17)
 
-- 
2.25.1

