From e7a62bf56d846fdb719e17e6e80c0eeb65c57aa0 Mon Sep 17 00:00:00 2001
From: Dengke Du <pinganddu90@gmail.com>
Date: Thu, 11 Sep 2025 11:25:46 +0800
Subject: [PATCH] tmp fix: ceres already build with suitespares

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7a80f16..fb9ceda 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,7 +36,7 @@ if(WIN32)
   set(Boost_USE_STATIC_LIBS FALSE)
 endif()
 find_package(Boost REQUIRED COMPONENTS ${BOOST_COMPONENTS})
-find_package(Ceres REQUIRED COMPONENTS SuiteSparse)
+find_package(Ceres REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(Lua REQUIRED)
 if(WIN32)
-- 
2.25.1

